neo/Ledger/MemoryPool.cs

```diff
@@ -102,6 +104,9 @@ public MemoryPool(NeoSystem system, int capacity)
         internal void LoadPolicy(Snapshot snapshot)
         {
             _maxTxPerBlock = (int)NativeContract.Policy.GetMaxTransactionsPerBlock(snapshot);
+            long newFeePerByte = NativeContract.Policy.GetFeePerByte(snapshot);
+            _policyChanged = newFeePerByte > _feePerByte;
```

Comment:
```suggestion
            _policyChanged = newFeePerByte != _feePerByte;
```

Reply:
There is no problem with price cuts, only the price increase will affect the verification results.